h5p-bildetema/semantics.json

```diff
@@ -270,5 +270,17 @@
         "type": "text"
       }
     ]
+  },
+  {
+    "label": "Default languages",
+    "name": "defaultLanguages",
+    "type": "list",
+    "entity": "Language",
+    "field": {
+      "description": "Language code should be on <a href=https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes>ISO-639-3</a> format",
+      "label": "Language Code",
+      "name": "languageCode",
+      "type": "text"
```

Comment:
Would maybe be a better user experience if the choosing of default language was a select field with only the possible languages to choose from. But that can be something we can do if they thing this is too difficult, since it will take some time to do